admin/admin.go

```diff
@@ -81,5 +121,26 @@ func setupRouter() *gin.Engine {
 	router.GET("/ready", func(c *gin.Context) {
 		c.String(http.StatusOK, "OK")
 	})
+
 	return router
 }
+
+func (m *RPCMethods) GetConfig(args *struct{}) *config.NetworkConfig {
+
+	m.Log.Info("GetConfig called")
+	return m.NetworkConfig
+}
+
+func (m *RPCMethods) GetL1Addresses(args *uint64) *map[string]string {
+	chain := filterByChainID(m.NetworkConfig.L2Configs, *args)
+	if chain == nil {
+		m.Log.Info("chain not found")
+		return nil
+	}
+	reply := map[string]string{
+		"L2CrossDomainMessenger": chain.L2Config.L1Addresses.L1CrossDomainMessengerProxy.String(),
+		"L2StandardBridge":       chain.L2Config.L1Addresses.L1StandardBridgeProxy.String(),
+	}
+	m.Log.Info("GetL2Addresses called", "chainId", *args)
```

Comment:
this prob should just be a debug log

m.Log.Debug("admin_getL2Addresses)

Reply:
done✅